crates/ra_parser/src/grammar/paths.rs

```diff
@@ -94,7 +94,7 @@ fn path_segment(p: &mut Parser, mode: Mode, first: bool) {
 
 fn opt_path_type_args(p: &mut Parser, mode: Mode) {
     match mode {
-        Mode::Use => return,
+        Mode::Use => {}
```

Comment:
Hm, not sure about this one: `return` contains more info, as it makes clear that noting happens down the line at all.